### DISCUSSION TITLE
Text wrap option [Jun 09]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2403] [Discord Threads]
> 🙋‍♂️ **cirsten**  
## Is there a wat to truncate what us displayed in a cell. For example in the 3rd row  (as well as the rest) I would prefer the "Comic Vine Link" cell to be cut off (truncated to the width of the cell ) rather that expanding the height (text wrap?)  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/984013787831083048/Capture.PNG)

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Not possible at the moment. You can raise an feature enhancement request here   
#### https://github.com/nocodb/nocodb/issues/new?assignees=&labels=&template=feature_request.md&title=%5BFeature%5D+  
#  
> 🙋‍♂️ **cirsten**  
#### will do  
#  
> 🙋‍♂️ **cirsten**  
#### can also ask if it possible to select exactly what colour you want assigned to single or multiple select field types? Or should I may this a feature request too?  
#  
> 💁 **o1lab**  
#### that is coming later  
#  
> 💁 **o1lab**  
#### we 're working on it  
#  
> 🙋‍♂️ **cirsten**  
#### great😀  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/984013788594462730)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
